doc/api/crypto.md

```diff
@@ -1217,6 +1217,16 @@ keys:
 
 All paddings are defined in `crypto.constants`.
 
+### crypto.timingSafeEqual(a, b)
+
+Returns true if `a` is equal to `b`, without leaking timing information that
+would allow an attacker to guess one of the values. This is suitable for
+comparing HMAC digests or secret values like authentication cookies or
+[capability urls](https://www.w3.org/TR/capability-urls/).
+
+A TypeError will be thrown if either `a` or `b` is not a Buffer instance,
+or if `a` and `b` have different lengths.
```

Comment:
I think the docs here should state that:
- `a` and `b` need to be `Buffer`s
- They need to have the same length.